atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunner.java

```diff
@@ -168,7 +168,7 @@ private SweepResults doRun(TableReference tableRef,
                 .batchSizeHint(batchConfig.candidateBatchSize())
                 .maxTimestampExclusive(sweepTs)
                 .shouldCheckIfLatestValueIsEmpty(sweeper.shouldSweepLastCommitted())
-                .timestampsToIgnore(sweeper.getTimestampsToIgnore())
+                .ignoreGarbageCollectionSentinels(!sweeper.shouldAddSentinels())
```

Comment:
Is this correct? If the sweeper should add sentinels, that suggests that it's doing conservative sweep, in which case it should likely ignore sentinels while sweeping.